.prettierrc.js

```diff
@@ -2,5 +2,4 @@ module.exports = {
   arrowParens: 'always',
   printWidth: 100,
   singleQuote: true,
-  trailingComma: 'es5',
```

Comment:
don't want trailing commas anymore? :(

Reply:
I still want es5 training commas but that is not even an option in Prettier 2. 😉 

> Change default value for trailingComma to es5 (#6963 by @fisker)

https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker

Reply:
Ohhh, nice!

Reply:
FYI: still an option, just default to `es5`, you can still set to `all` `none`.

Reply:
@fisker Thanks for pointing that out. Yeah, technically it's still an option and I removed this setting because we want to use the default value for this option.